llbase/llidl.py

```diff
@@ -36,8 +11,7 @@
 import uuid
 from functools import total_ordering
 
-from . import llsd
-from . import lluuid
+from llbase import llsd
```

Comment:
Why not just import from llsd?

Reply:
`llbase.llsd` is a bit of a compatibility shim in that it re-exports a few things that the llsd package's base module does not. Specifically, some of the contents of `llsd.base` like `is_string`, `StringTypes`, `is_bytes`, etc.